src/main/java/org/example/intro/config/OpenApiConfig.java

```diff
@@ -0,0 +1,23 @@
+package org.example.intro.config;
+
+import io.swagger.v3.oas.models.Components;
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.security.SecurityRequirement;
+import io.swagger.v3.oas.models.security.SecurityScheme;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class OpenApiConfig {
+
+  @Bean
+  public OpenAPI customOpenApi() {
+    return new OpenAPI()
+      .components(new Components().addSecuritySchemes("BearerAuth",
```

Comment:
create constant for BearerAuth
check whitespaces (maybe I am wrong, but looks strange)

Reply:
++, thank, updated